src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+body {
+  font-family: ' Avenir ', sans-serif;
+  font-weight: 300;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: ' Avenir ';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+.search-box {
+  position: relative;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+}
+
+.search-box--big {
+  height: 70px;
```

Comment:
Ensure that there is no requirement for a fixed size for `.search-box--big`. If the content can dictate the size, avoid setting a fixed height.